pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -161,7 +171,8 @@ func (a *AgentClusterInstall) Generate(dependencies asset.Parents) error {
 
 		if agentClusterInstall.Spec.PlatformType == hiveext.ExternalPlatformType {
 			agentClusterInstall.Spec.ExternalPlatformSpec = &hiveext.ExternalPlatformSpec{
-				PlatformName: installConfig.Config.Platform.External.PlatformName,
+				PlatformName:           installConfig.Config.Platform.External.PlatformName,
+				CloudControllerManager: hiveext.CloudControllerManager(installConfig.Config.Platform.External.CloudControllerManager),
```

Comment:
I don't think we need to set this in addition to the install-config override for the purposes of this bug. Even for [AGENT-704](https://issues.redhat.com/browse/AGENT-704) the install-config override would suffice; adding this field would only be for the benefit of assisted.